mmv1/templates/terraform/custom_expand/bigquery_table_ref.go.tmpl

```diff
@@ -40,5 +40,19 @@ func expand{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d tpgresource.T
 		transformed["tableId"] =  parts[3]
 	}
 
+	configError := "Invalid BigQuery job configuration. You must either:\n" +
```

Comment:
How about "Invalid BigQuery job destination_table configuration" to be more explicit